### PR TITLE
Move load() calls to top of WORKSPACE

### PR DESCRIPTION
**Pull request by buildbreaker**
https://github.com/lyft/envoy-mobile/issues/79

https://github.com/lyft/envoy-mobile/pull/66#discussion_r292604149

Signed-off-by: Alan Chiu <achiu@lyft.com>

For an explanation of how to fill out the fields, please see the relevant section
in [PULL_REQUESTS.md](https://github.com/envoyproxy/envoy/blob/master/PULL_REQUESTS.md)

Description: move load methods to top of `WORKSPACE`
Risk Level: low
Testing: n/a
Docs Changes: n/a
Release Notes: n/a
[Optional Fixes #Issue]
[Optional Deprecated:]

### CODE REVIEW

**Comment by rebello95** [comment]
CI should pass after rebasing

File: WORKSPACE [hunk 1]
```diff
@@ -1,5 +1,15 @@
 load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")
 load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+load("@rules_foreign_cc//:workspace_definitions.bzl", "rules_foreign_cc_dependencies")
```
Comment (rebello95) [inline WORKSPACE]: Can we alphabetize these?

**Comment by buildbreaker** [comment]
We aren't able to have the `load()` methods all on top like in a `BUILD` file because we're required to use `http_archive()` and `git_repository()` prior to `load()`.

Additionally, this isn't that conventional as we initially thought as https://github.com/bazelbuild/bazel/ also clusters these items by context: https://github.com/bazelbuild/bazel/blob/master/WORKSPACE.

Other examples:
https://github.com/bazelbuild/rules_kotlin/blob/master/WORKSPACE
https://github.com/bazelbuild/rules_kotlin/blob/master/WORKSPACE

**Comment by buildbreaker** [comment]
Going to close as this is a marginal stylistic improvement and it is actually not as commonly practiced

**Pull request closed without merging**